docs/Configuration/yaml-files.md

```diff
@@ -767,6 +767,8 @@ org_settings:
 
 #### failing_policies_webhook
 
+> The below settings is for enabling webhooks on policies at the 'All Team' level. To enable this automation on policies in a Team, nest these settings under `team_settings`. 
```

Comment:
Let's make this a bit more concise:
```suggestion
> These settings can also be configured per-team when nested under `team_settings`. 
```
(I foresee this being the sort of note we end up pasting in other places)